chapters/data/main.tex

```diff
@@ -72,293 +85,290 @@ \subsection{Data structure}\label{subsec:structure}
-    associated with it a summed absolute correlation coefficient, \(c_j\), given
-    by:
-    
-    \[
+    Consider a dataset with \(m \in \mathbb{N}\) columns,
+    \(A = \left\{A_1, \ldots, A_m\right\}\). Attribute \(A_j\) has
+    associated with it a \emph{summed absolute correlation coefficient},
+    \(c_j\), given by:
+    \begin{equation}\label{eq:abs_corr}
         c_j = \sum_{k=1}^{m} \left\| \rho_{A_j, A_k} \right\|
-    \]
+    \end{equation}
 
     Here, \(\rho_{A_j, A_k}\) is the Pearson's correlation coefficient
     between attributes \(A_j\) and \(A_k\).
-\end{definition}\label{def:absolute_correlation}
+\end{definition}
 
-\begin{figure}[htbp]
-    \makebox[\textwidth]{%
-        \centering
-        \includegraphics[height=.6\paperheight]{corr_heatmap/with_nums.pdf}
+\begin{figure}
+    \resizebox{\textwidth}{!}{%
+        \includegraphics[width=\linewidth]{corr_heatmap.pdf}
     }
-    \caption{A heat map of the pairwise correlation coefficients for the key
-        cost attributes. The attributes have been ordered according to their
-        summed absolute correlation coefficient.}%
-    \label{fig:corr_heatmap}
+    \caption{%
+        Pairwise correlation coefficients for the key cost attributes
+    }\label{fig:corr_heatmap}
 \end{figure}
 
 Upon inspection of the heat map, there are many cost components that have no
-significant linear correlation with any of the other attributes. Despite this,
-however, there are clear correlations between many of the attributes; some of
-these are easier to realise than others.
+substantial linear correlation with any of the other attributes. The absence of
+correlation here only adds to the evidence that the patients present in the data
+present themselves to hospital with a wide array of needs. Having said that,
+there are clear correlations between several of the attributes; some of these
+are easier to realise than others.
 
 For instance, ignoring the main diagonal, the largest value is that between
-total costs (COST) and net costs with a value of 0.94. This indicates almost
-total positive linear correlation between these two variables, and that makes
-sense given that the net cost of a spell is just the total cost corrected for a
-number of reimbursable costs like critical care (CRIT) and non-contracted income
-(NCI) which are entered as negative values in the dataset \-- hence their
-distinctly negative correlation coefficients with the other variables.
-Typically, these deductible costs are small (see Table~\ref{tab:summative}) so a
-strong correlation between costs and net costs is to be expected.
-
-Another example is the strong correlation amongst the length of stay
-(TRUE\_LOS), and ward and overhead costs (WARD and OVH respectively). These are
-well-known relationships that can be justified anecdotally: the longer a patient
+total costs (COST) and net costs (NetCost) with a value of 0.94. This high value
```

Comment:
Do you want to put variable names like these in a different font? Perhaps `\texttt{COST}` ?

Reply:
That's a good idea. I'll go through and pick those up.

Reply:
Fixed by bec4e1d